docs/releases/3.2.txt

```diff
@@ -413,6 +413,9 @@ Tests
   <django.db.transaction.on_commit>` in a list. This allows you to test such
   callbacks without using the slower :class:`.TransactionTestCase`.
 
+* :meth:`.TransactionTestCase.assertQuerysetEqual` now supports querysets
+  comparison.
+
```

Comment:
Just reading that in isolation, I think, _"Well what did it do before?"_ 

Can we say a little bit more? 

`…now supports direct querysets comparison, rather than requiring comparison against a string representation.`

(Not 100% happy with that.)

Reply:
Maybe:
```
…now supports direct comparison against another queryset rather than being restricted to comparison against a list of string representations of objects when using the default value for the `transform` argument.
```